Resources/Prototypes/Entities/Objects/Specific/chemical-containers.yml

```
@@ -77,7 +77,7 @@
       thresholds:
       - trigger:
           !type:DamageTrigger
-          damage: 200
+          damage: 240 # Carpmosia-edit - extendedReagentStorage
```

This was probably done by mistake thru find & replace, unrelated to capacities.
```suggestion
          damage: 200
```

---

Whoops yeah, I shoulda caught that myself, sorry.